bigtable-dataflow-parent/bigtable-beam-import/pom.xml

```diff
@@ -334,19 +422,6 @@ limitations under the License.
       </testResource>
     </testResources>
     <plugins>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-enforcer-plugin</artifactId>
-        <executions>
-          <execution>
-            <id>enforce</id>
-            <configuration>
-              <skip>true</skip>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-
       <plugin>
         <artifactId>maven-jar-plugin</artifactId>
         <configuration>
```

Comment:
can't comment on the line directly since it's not in the diff, but you can remove the `build-helper-maven-plugin`